nomic-bot/src/universe/commands/happy.js

```diff
@@ -0,0 +1,24 @@
+import { addCommand, updatePlayer, createComment } from './index';
+import moment from 'moment';
+import { chain, startsWith } from 'lodash';
+
+
+addCommand('/happy', (_, { playerComments, player }) => {
+    const now = moment();
+    const latestHappyCommentWeeks = chain(playerComments)
+        .filter(comment => startsWith(comment.body, '/happy'))
+        .map(comment => now.diff(moment(comment.created_at), 'week'))
+        .min()
+        .value()
+
+    if (latestHappyCommentWeeks === undefined || latestHappyCommentWeeks >= 1) {
+        player.village.happiness += 1;
```

Comment:
It's weird to me you didn't use ++ here. Also, I have a general question about these commands. Is it reasonable to assume a player has a village? I vaguely remember something about active players needing to have a village. But what would happen if someone tried to use this command and they don't have a village? I'm just hoping that wouldn't crash the bot.